code/modules/mob/living/simple_animal/hostile/pulse_demon/pulsedemon.dm

```diff
@@ -121,7 +122,28 @@
 	if(statpanel("Status"))
 		stat(null, text("Charge stored: [charge]W"))
 		stat(null, text("Max charge stored: [maxcharge]W"))
-		
+		stat(null, text("Health: [health]/[maxHealth]"))
+		stat(null, text("Draining power sources: [draining ? "Yes" : "No"]"))
+		stat(null, text("Drain rate: [charge_absorb_amount]"))
+		stat(null, text("APC takeover time: [takeover_time] seconds"))
+
+/mob/living/simple_animal/hostile/pulse_demon/proc/determine_name()
```

Comment:
I get what you're going for, although this does mean you can never bus a pulse demon with a special name because it will update